test/functional/test_ipmi_console.py

```diff
@@ -66,6 +66,36 @@ def reset_console(channel, timeout=10):
             raise RuntimeError('ipmi-console reset expires {}s'.
                                format(timeout))
 
+class test_ipmi_start_stop(unittest.TestCase):
+    node_name = "default"
+    node_workspace = os.path.join(config.infrasim_home, node_name)
+
+    def test_start_stop(self):
+        os.system("sudo ipmi-console stop")
+        os.system("sudo infrasim node destroy")
+        os.system("rm -rf {}".format(self.node_workspace))
+        os.system("pkill socat")
+        os.system("pkill ipmi")
+        os.system("pkill qemu")
```

Comment:
Trust other test's setup and teardown. Don't have to do above stop/destroy/pkill here. Instead, it's better to be put as a `teardown` of this `class test_ipmi_start_stop`